app/editor/src/features/content/form/ContentClipForm.tsx

```diff
@@ -37,7 +37,7 @@ export const ContentClipForm: React.FC<IContentClipFormProps> = ({
   path: initPath,
   setClipErrors,
 }) => {
-  const { values, setFieldValue } = useFormikContext<IContentForm>();
+  const { values, setFieldValue } = useFormikContext<IContentForm>() ?? {};
```

Comment:
We should not do this.  All you're doing it hiding errors from the linter.

Reply:
Updated, thanks